dev/src/generate/binding_python.rs

```diff
@@ -17,9 +17,86 @@
 
 use crate::generate::parser::Services;
 use anyhow::Result;
+use itertools::Itertools;
+use std::fs;
+use std::path::PathBuf;
 
-pub fn generate(services: &Services) -> Result<()> {
-    println!("{:?}", services);
+use super::parser::ConfigType;
+
+fn enabled_service(srv: &str) -> bool {
```

Comment:
Consider use `cargo_metadata` to get the enabled feature list? https://docs.rs/cargo_metadata/latest/cargo_metadata/struct.Node.html#structfield.features

Reply:
Sure.